data/build.gradle.kts

```
@@ -16,6 +16,8 @@ dependencies {
     implementation("com.squareup.retrofit2:converter-gson:2.9.0")
     implementation("com.squareup.okhttp3:okhttp:4.10.0")
     implementation("com.squareup.okhttp3:mockwebserver:4.10.0")
+    implementation( "com.squareup.okhttp3:logging-interceptor:4.10.0")
+    implementation("com.google.dagger:hilt-core:$hiltVersion")
```

```suggestion
    implementation("com.google.dagger:hilt-core:$hiltVersion")
    kapt("com.google.dagger:hilt-android-compiler:$hiltVersion")
```
힐트를 사용하면 빌드 에러가 발생하는 경우를 종종 만나게 됩니다.
이런 경우에는 가장 먼저 hilt 관련 디펜던시를 잘 적용했는지를 확인해보시고, 그 이후에는 힐트 관련 모듈을 만들때 빠진 것이 없는지 확인해보시면 좋습니다.